test.js

```diff
@@ -0,0 +1,195 @@
+'use strict';
+
+var async = require('async');
+var spawn = require('child_process').spawn;
+var docker = new require('dockerode')();
+var http = require('http');
+
+async.waterfall([
+  cleanupInstances,
+  createNewInstance,
+  sleep(2000),
+  activateInstanceLicense,
+  sleep(2000),
+  initializeInstanceDatabase,
+  sleep(2000),
+  waitForNewInstance,
+  createTestDatabase,
+  runTests,
+], function(testErr) {
+  cleanupInstances(function(cleanupErr) {
+    if (cleanupErr) {
+      console.error('error cleaning up:', cleanupErr);
+    }
+    if (testErr) {
+      console.error('error running tests:', testErr);
+      process.exit(1);
+    }
+  });
+});
+
+function sleep(n) {
+  return function(container, next) {
+    setTimeout(next, n, null, container);
+  };
+}
+
+function createNewInstance(next) {
+  console.log('pulling image...');
+  docker.pull('ibmcom/cloudant-developer:1.0.1', function(err, stream) {
+    docker.modem.followProgress(stream, function(err, output) {
+      if (err) {
+        return next(err);
+      }
+      console.log('starting container...');
+      docker.createContainer({
+        Image: 'ibmcom/cloudant-developer:1.0.1',
+        name: 'cloudant-testdb',
+        HostConfig: {
+          PortBindings: {
+            '80/tcp': [{HostPort: '8080'}],
+          },
+          Privileged: true,
```

Comment:
That's for `:latest` and `:2.0.0`, but this is still using the `:1.0.1` generation.